### DISCUSSION TITLE
优化的一些想法！

### DISCUSSION THREAD

**Discussion by Yanranh** [qier222/YesPlayMusic#1000] [Ideas]
在HelloGithub上看到你们一起开发的轻量级音乐播放器，YesPlayMusic。下载使用后，真的觉得，YES，PLAY MUSIC!
 嗯就是它了！设计的理念和功能都满足了我理想中的音乐播放器，界面简洁，保留了最常用的功能，最最最好的是，因为引入了外源，之前歌单里灰色的歌曲又能听的感觉真的是太nice了！
  简单看了一下功能介绍，反映一下优化使用方面的小小想法吧。就是在点击进入我喜欢的音乐时，想翻一下以前收藏的歌曲，
![image](https://user-images.githubusercontent.com/59815458/138038663-131cb056-daf5-4f3c-992d-1979b3e50914.png)

点击目标歌曲左边的专辑封面，可以进到歌曲页面。
![image](https://user-images.githubusercontent.com/59815458/138038711-ea24df0a-fed2-4af8-9171-4403c707f308.png)
但是重新回到选择歌单页面的时候，页面又重新载入，没法保留我先前下拉到的目标歌曲地方，
![image](https://user-images.githubusercontent.com/59815458/138038756-44314a19-1927-488e-9829-d8a6c5fa2278.png)

感觉这里可以放一个保留定向，回到歌单页面时，能看到下拉到的原页面。

**Comment by qier222** [comment]
这个是存在了很久的问题，目前还没想到好的解决办法

**Comment by 762556500** [comment]
YesPlayMusic不能识别本地音乐吗？现在

**Comment by ZZZZZZZZZZZZM** [comment]
这是我感觉很棒的音乐播放器，不知后续是否考虑会加入桌面歌词这一功能

**Reply by memorydream** [reply]
增加过，又拿掉了
